keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/diff/DualNumber.java

```diff
@@ -122,4 +122,41 @@ public DualNumber pow(double value) {
     public DualNumber unaryMinus() {
         return times(-1.0);
     }
+
+    public DualNumber pow(DualNumber exponent) {
```

Comment:
This is already defined in the `powerTo` method above, no need to have it twice. Either rename the one above to `pow` or do a pass through like:

    public DualNumber pow(DualNumber exponent) {
        return powerTo(exponent);
    }